package.json

```diff
@@ -32,12 +41,13 @@
     "karma-tap-pretty-reporter": "3.0.5",
     "karma-webpack": "2.0.4",
     "kw-doc": "1.1.2",
-    "kw-web-suite": "6.0.1",
+    "kw-web-suite": "6.0.2",
     "resemblejs": "2.2.6",
     "tap-markdown": "1.2.1",
     "tap-spec": "4.1.1",
     "tape": "4.8.0",
     "tape-catch": "1.0.6",
+    "webpack-cli": "2.0.14",
```

Comment:
That's part of `kw-web-suite` already

Reply:
I was able to do a clean install and build by removing the `package-lock.json` file and the `nodes-modules` directory.

However, I get those npm warnings:
```node
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update! 

> fsevents@1.1.3 install /Users/agirault/Projects/vtk-js/node_modules/fsevents
> node install

[fsevents] Success: "/Users/agirault/Projects/vtk-js/node_modules/fsevents/lib/binding/Release/node-v59-darwin-x64/fse.node" is installed via remote

> spawn-sync@1.0.15 postinstall /Users/agirault/Projects/vtk-js/node_modules/spawn-sync
> node postinstall

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN karma-webpack@2.0.4 requires a peer of webpack@^1.0.0 || ^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN resemblejs@2.2.6 requires a peer of canvas@^1.6.5 but none is installed. You must install peer dependencies yourself.
npm WARN webpack-dev-middleware@1.12.2 requires a peer of webpack@^1.0.0 || ^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
```

Any we could try to address?

Reply:
Ok I'm able to do a build now, removing `webpack-cli`. Forgot to remove package-lock.json.

Those peer warnings are mostly harmless. `karma-webpack` could be updated to 3.0.0 (to use webpack 4), and `webpack-dev-middleware` can be bumped up to 3.1.2 (also to use webpack 4). Those are pretty major version jumps. @jourdain do we want to update?

Reply:
Yes if that does not break anything (tests...)